client-src/elements/chromedash-feature-detail.js

```diff
@@ -156,7 +156,10 @@ class ChromedashFeatureDetail extends LitElement {
       .inline-list {
         display: inline-block;
         padding: 0;
-        margin: 0;
+      }
+
+      .inline-list li + li {
+        margin-top: 2px;
       }
```

Comment:
I guess these changes are not needed if margin added into `.feature-link sl-tag::part(base)`.